experimental/PropertyDDS/packages/property-dds/package.json

```diff
@@ -61,7 +61,7 @@
     "@fluidframework/local-driver": ">=2.0.0-internal.1.2.1 <2.0.0-internal.2.0.0",
     "@fluidframework/mocha-test-setup": ">=2.0.0-internal.1.2.1 <2.0.0-internal.2.0.0",
     "@fluidframework/sequence": ">=2.0.0-internal.1.2.1 <2.0.0-internal.2.0.0",
-    "@fluidframework/server-local-server": "^0.1038.1000",
+    "@fluidframework/server-local-server": "^0.1037.2001",
```

Comment:
i think you want ^0.1037.2000 which will pick up 0.1037.2001. i'm not totally confident npm care if the last digit is zero or not, but i think that's what the semver spec defines

Reply:
Agree, likely it should not matter unless you have env that's can fulfill through installed/locked 1037.2000. I'll double check this, but it should be fine to enforce specific min patch here.